src/popup/bookmark.js

```diff
@@ -160,7 +160,8 @@ App.registerPanel(P_SAVING, {
 
     const headers = new Headers({
       "Content-Type": "application/vnd.api+json",
-      "Accept": "application/vnd.api+json"
+      "Accept": "application/vnd.api+json",
```

Comment:
my guess is these content/accept headers are going to be pretty consistent in your use of the api, should they be constantized at some higher level?

Reply:
I _GUESS_ SO but that might be better suited for another PR

Reply:
Actually, I'm going to refactor this into a better "api" ... api in a different PR so that all the comms is in one spot